### PR TITLE
Pre define workflow for Sock-Shop catalogue-pod-delete  

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: Oum Kale <oum.kale@mayadata.io>

This PR has argowf-chaos-catalogue-pod-delete-cron-wf and argowf-chaos-catalogue-pod-delete workflow

### CODE REVIEW

File: workflows/sock-shop-demo/workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        command: [sh, -c]
+        args:
+          [
+            "kubectl apply -f /tmp/pod-delete.yaml -n {{workflow.parameters.adminModeNamespace}}",
+          ]
+    
+    - name: load-test
+      container:
+        image: litmuschaos/litmus-app-deployer:latest
+        args: ["-namespace=loadtest"]
```
Comment (ispeakc0de) [inline workflows/sock-shop-demo/workflow.yaml:117]: Are you tuning load-test or sock-shop run based on the value of namespace? better to have a relevant flag/decider name.

File: workflows/sock-shop-demo/workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                              value: '10'
+                
+                            # pod failures without '--force' & default terminationGracePeriodSeconds
+                            - name: FORCE
+                              value: 'false'
+      container:
+        image: litmuschaos/litmus-checker:latest
+        args: ["-file=/tmp/chaosengine.yaml","-saveName=/tmp/engine-name"]              
+
+    #- name: revert-chaos
```
Comment (ispeakc0de) [inline workflows/sock-shop-demo/workflow.yaml]: you can uncomment this code, whenever done with debugging.

File: workflows/sock-shop-demo/workflow.yaml (hunk 1 above)
Comment (oumkale) [reply workflows/sock-shop-demo/workflow.yaml:117]: May I use a flag like a bool type? 
Is that ok to have a namespace flag only? It is also working as a flag only.

File: workflows/sock-shop-demo/workflow.yaml (hunk 2 above)
Comment (oumkale) [reply workflows/sock-shop-demo/workflow.yaml]: Done

File: workflows/sock-shop-demo/workflow.yaml (hunk 1 above)
Comment (ispeakc0de) [reply workflows/sock-shop-demo/workflow.yaml:117]: deployerType/typeName flag can be used, which can take [weak, resilient,load-test] values and test-tool code can select the corresponding file path based on this flag?
namespace flag can still be there. You can set some default value based on usecase(sock-shop for sock-shop and loadtest for load-test)

Comment (ksatchit) [reply workflows/sock-shop-demo/workflow.yaml:117]: We have merged the PR on test-tools: https://github.com/litmuschaos/test-tools/pull/145 that is dependent on the namespace name to launch the right manifest. This is something we should change in the next release. Resolving for now.

**Review by ispeakc0de — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**